static/docs/user-guide/experiments/dirs.md

```diff
@@ -0,0 +1,97 @@
+# How to Manage Experiments by Directories
+
+Using a separate directory for each experiment is the most intuitive solution
+for managing experiments and is the first thing that comes to mind. Most of
+DS/ML practitioners are already familiar with this approach.
+
+<p align="center">
+<img src="/static/img/user-guide/experiments/dirs.png" />
+</p>
+
+This approach is most suitable when the different experiments that are being
+managed do not have significant differences in their code or the pipeline, but
+maybe change on the input datasets, processing parameters, configuration
+settings, etc.
+
+Often it is possible to generate these experiment directories automatically (or
+almost automatically) from the code of the main project (using the parameters or
+configuration settings), so keeping them in Git is not interesting or useful.
+What we would like to track instead are just the parameters that were used to
+generate the experiment directory and the results of the evaluation (metrics),
+so that we can figure out which parameters give the best results.
+
+## Examples
+
+There is a very basic example of using directories for each experiment at the
+end of
+[this interactive tutorial](https://katacoda.com/dvc/courses/basics/pipelines).
+
+## How it works
+
+If we have a directory named `experiment1/` which contains the pipeline of the
```

Comment:
except the pipeline - what else does this directory contain? what about code? do we copy the whole project?

Reply:
I think this is too project specific and it is impossible to explain such details on a section like this.
The best that can be done is to have a concrete example somewhere else and to link it from the section of the examples.

Reply:
there are a few recommendations we can make - clean data before copying? use a simple JSON conf? use DVC-files that are setup properly to be relative.

Those ^^ are general problems. Without giving some hints at least it's not very actionable. People who can come with a solution to those problems don't even probably need a section like this (and I can send you a few names).

Reply:
> there are a few recommendations we can make - clean data before copying? use a simple JSON conf? use DVC-files that are setup properly to be relative.

All these seem common sense to me. Anyway I don't see how they can be explained without having an example as a reference.
Maybe someone can give it a try and let's see whether they make sense.

Reply:
Since there is no understanding from our end on how to give a guidance let's remove this page then for now and wait until the ticker on DVC core is resolved and wait for someone to take the #159.

Reply:
> All these seem common sense to me.

If they are common sense why would it be so hard to explain/mention them?